### DISCUSSION TITLE
Unload Combine when raining

### DISCUSSION THREAD

**Discussion by tslogick-1** [Courseplay/Courseplay_FS22#2515] [Q&A] [answered]
Is anyone else having this issue?

Courseplay latest version

Harvesting during raining and using the override in courseplay

The combine will stop and call for the unloader, extend the pipe, but when the unloader drives under the pipe the combine never starts unloading the grain.

**Comment by Tensuko** [comment, marked as answer]
Fixed in 7.2.1.4
